plugins/modules/sp_configure.py

```diff
@@ -8,20 +8,20 @@
 DOCUMENTATION = r'''
 ---
 module: sp_configure
-short_description: Make instance level system configuration changes via sp_configure.
+short_description: Make instance level system configuration changes via C(sp_configure)
 description:
-     - Read instance level system configuration for a given configuration and update to a new value as provided.
-     - If the configuration needs a restart, a warning message will be returned stating a restart is required for the new value to be used.
+- Read instance level system configuration for a given configuration and update to a new value as provided.
+- If the configuration needs a restart, a warning message will be returned stating a restart is required for the new value to be used.
```

Comment:
Enhancement recommendation (not this PR): the return data should include a boolean field on whether a restart is required, so that the restart can optionally be handled in subsequent Ansible tasks.

Reply:
https://github.com/lowlydba/lowlydba.sqlserver/issues/120